group-coordinator/src/test/java/org/apache/kafka/coordinator/group/OffsetMetadataManagerTest.java

```diff
@@ -1834,6 +1834,44 @@ public void testFetchOffsetsWithUnknownGroup() {
         assertEquals(expectedResponse, context.fetchOffsets("group", request, Long.MAX_VALUE));
     }
 
+    @Test
+    public void testFetchOffsetsWithTopicIds() {
+        Uuid fooId = Uuid.randomUuid();
+        Uuid barId = Uuid.randomUuid();
+        OffsetMetadataManagerTestContext context = new OffsetMetadataManagerTestContext.Builder().build();
+
+        context.groupMetadataManager.getOrMaybeCreatePersistedConsumerGroup("group", true);
+
+        context.commitOffset("group", "foo", 0, 100L, 1);
+        context.commitOffset("group", "bar", 0, 200L, 1);
```

Comment:
nit: Should we extract `foo`, `bar`, and `group` as variables?

Reply:
This is not necessary in my opinion.